CI is red because of the scripting plublishing nightly docs

https://travis-ci.org/ckeditor/ckeditor5/builds/479475326#L1105

![image](https://user-images.githubusercontent.com/156149/51250404-8fcee300-1996-11e9-8710-c579ef6ba4ad.png)

Something weird happens there.

```bash
node ./scripts/docs/build-docs.js --production
JSDoc started...
No errors found during the validation.
Documented 410 files!
(node:6987) UnhandledPromiseRejectionWarning: Error: Cannot find module 'hexo'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/travis/build/ckeditor/ckeditor5/node_modules/umberto/hexo/hexo-shim.js:8:18)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
(node:6987) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:6987) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Done in 21.88s.
```

That's the reason why this script stopped working.

--- a/.travis.yml
+++ b/.travis.yml
@@ -24,12 +24,11 @@ install:
 - rm -rf node_modules
 - yarn install
 script:
-- yarn run docs --production
-#- yarn run test --reporter=dots
-#- yarn run docs:api --validate-only
-#- 'if [ $TRAVIS_TEST_RESULT -eq 0 ]; then
-#    travis_wait yarn run docs:build-and-publish-nightly;
-#  fi'
+- yarn run test --reporter=dots
+- yarn run docs:api --validate-only
+- 'if [ $TRAVIS_TEST_RESULT -eq 0 ]; then
+    travis_wait yarn run docs:build-and-publish-nightly;
+  fi'
 notifications:
   slack:
     rooms:

--- a/.travis.yml
+++ b/.travis.yml
@@ -18,9 +18,10 @@ before_install:
 - sh -e /etc/init.d/xvfb start
 - npm i -g yarn
 install:
-- yarn add mgit2 --ignore-workspace-root-check
+- npm i mgit2 --no-save
 - mgit sync --resolver-url-template="https://github.com/\${ path }.git"
-- yarn
+- rm -rf node_modules
+- yarn install
 script:
 - yarn run test --reporter=dots
 - yarn run docs:api --validate-only

--- a/.travis.yml
+++ b/.travis.yml
@@ -24,11 +24,12 @@ install:
 - rm -rf node_modules
 - yarn install
 script:
-- yarn run test --reporter=dots
-- yarn run docs:api --validate-only
-- 'if [ $TRAVIS_TEST_RESULT -eq 0 ]; then
-    travis_wait yarn run docs:build-and-publish-nightly;
-  fi'
+- yarn run docs --production
+#- yarn run test --reporter=dots
+#- yarn run docs:api --validate-only
+#- 'if [ $TRAVIS_TEST_RESULT -eq 0 ]; then
+#    travis_wait yarn run docs:build-and-publish-nightly;
+#  fi'
 notifications:
   slack:
     rooms:

--- a/.travis.yml
+++ b/.travis.yml
@@ -18,8 +18,9 @@ before_install:
 - sh -e /etc/init.d/xvfb start
 - npm i -g yarn
 install:
-- npm i mgit2 --no-save
+- yarn add mgit2 --ignore-workspace-root-check
 - mgit sync --resolver-url-template="https://github.com/\${ path }.git"
+- git checkout package.json yarn.lock
 - rm -rf node_modules
 - yarn install
 script:

